jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBQueryResultSet.java

```diff
@@ -66,6 +67,7 @@
   private final String TIMESTAMP_STR = "Time";
   private static final String limitStr = "LIMIT";
   private static final String offsetStr = "OFFSET";
+  private static final int firstColumnTypeIndex = 2;
```

Comment:
Constant naming should be like this :   LIMIT_STR

Reply:
Did you mean also change other constant names like `limitStr` and `offsetStr` to `LIMIT_STR` and `OFFSET_STR`?

Reply:
> Constant naming should be like this : LIMIT_STR

Thank you. Fixed